custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -92,16 +92,16 @@ def _state_aggregation_query(self, state_id):
         alive_in_month = "(child_health.date_death IS NULL OR child_health.date_death - {} >= 0)".format(
             start_month_string
         )
-        migration_status = (
+        migrated = (
             "agg_migration.is_migrated IS DISTINCT FROM 0"
             "AND agg_migration.migration_date::date < {start_month_string}"
         ).format(start_month_string=start_month_string)
-        registered_status = (
-            "agg_availing.is_registered IS DISTINCT FROM 1"
+        not_registered = (
+            "agg_availing.is_registered = 0"
             "AND agg_availing.registration_date::date < {start_month_string}"
         ).format(start_month_string=start_month_string)
-        seeking_services = "NOT {registered_status} AND NOT {migration_status}".format(
-            registered_status=registered_status, migration_status=migration_status)
+        seeking_services = "NOT {not_registered} AND NOT {migrated}".format(
```

Comment:
the double negative is a bit confusing "not not registered". It might be more straightforward to have the indicator be the opposite